cannon/testdata/example/alloc/go.mod

```diff
@@ -7,8 +7,8 @@ toolchain go1.21.1
 require github.com/ethereum-optimism/optimism v0.0.0
 
 require (
-	golang.org/x/crypto v0.26.0 // indirect
```

Comment:
This seems like unnecessary diff?

Reply:
It is forced because of the op-geth update

Reply:
Geth rc.4 landed on monorepo develop branch, this diff is now dropped